main.tf

```diff
@@ -28,6 +28,8 @@ resource "castai_node_configuration" "this" {
     dns_cluster_ip       = try(each.value.dns_cluster_ip, null)
     instance_profile_arn = try(each.value.instance_profile_arn, null)
     key_pair_id          = try(each.value.key_pair_id, null)
+    volume_type          = try(each.value.volume_type, null)
```

Comment:
It would be nice to also update the https://github.com/castai/terraform-castai-eks-cluster/blob/main/README.md with those new fields.

Reply:
It is a known issue we hit several times before. As node_configurations is a map of type any https://github.com/castai/terraform-castai-eks-cluster/blob/main/variables.tf#L64 there is no easy way to describe possible fields in the docs